Occasionally, nvim's `:trust` confirmation messages interfere with fennel output to cause some tests fail

```log
Failure → /usr/local/share/lua/5.1/vusted/run.lua @ 27
command :Fnl with range set to % but not in a fennel buffer should ignore current buffer
Random seed: 3424245346
/usr/local/share/lua/5.1/vusted/run.lua:27: Expected objects to be equal.
Passed in:
(string) '/home/runner/work/nvim-thyme/nvim-thyme/test/context/.config/nvim/.nvim-thyme.fnl is not trusted.
[i]gnore, (v)iew, (d)eny, (a)llow: 
nil'
Expected:
(string) 'nil'

stack traceback:
	/usr/local/share/lua/5.1/vusted/run.lua:27: in function </usr/local/share/lua/5.1/vusted/run.lua:1>
	[string ":lua"]:10: in main chunk
[i]gnore, (v)iew, (d)eny, (a)llow: :(+ 1 2)="Fnl (+ 1 2)":Fnl (+ 1 2)...:Fnl (+ 1 2):(+ 1 2)="Fnl (+ 1 2)":Fnl (+ 1 2)...:Fnl (+ 1 2):(+ 1 2):(+ 1 2)="Fnl (+ 1 2)":Fnl (+ 1 2)3:(+ 1 2):(+ 1 2)="Fnl (+ 1 2)":Fnl (+ 1 2)3make: *** [Makefile:116: test] Error 1
Error: Process completed with exit code 2.
```
https://github.com/aileot/nvim-thyme/actions/runs/18595153225/job/53019410198#step:7:176

https://github.com/aileot/nvim-thyme/actions/runs/18595209969/job/53019605805?pr=104#step:7:200

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -39,7 +39,7 @@ jobs:
       - name: Run tests
         env:
           VUSTED_EXTRA_FLAGS: "--defer-print --suppress-pending"
-          VUSTED_EXTRA_ARGS: ""
+          VUSTED_EXTRA_ARGS: "-Es"
         run: |
           make -j build
           make test

